<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorClass</class>
 <widget class="QMainWindow" name="MonitorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="GaugePanel" name="panelMagTrans16" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelFast" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelMagTrans13" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelMagTrans14" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelFeshbach" native="true">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>120</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelLevitation" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelQp" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelZeemanSlower" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelMagTransDown" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panelMagTransUp" native="true">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>370</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="GaugePanel" name="panel2dPush" native="true">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>370</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
   </widget>
   <widget class="RulerTemp" name="tempMagTrans1" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>650</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="RulerTemp" name="tempMagTrans2" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>650</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="RulerTemp" name="tempMagTrans3" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>650</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="RulerTemp" name="tempMagTransUp" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>650</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="ToggleSwitch" name="toggleSwitchRun" native="true">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>180</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelForScienceCell">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>650</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Science Cell</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScienceCell">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>690</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelFor3DMOT">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>650</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>3D MOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3DMOT">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>690</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelFor2DMOT">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>790</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>2D+ MOT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2DMOT">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>830</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelForZeemanSlower">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>790</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zeeman Slower</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelZeemanSlower">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>830</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labMonitorTitle">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>851</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>费米子实验室监控系统</string>
    </property>
   </widget>
   <widget class="ToggleSwitch" name="toggleSwitchAlarm" native="true">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>300</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetScienceCell">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSet3DMOT">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSet2DMOT">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>800</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置阈值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSetZeemanSlower">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>800</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置阈值</string>
    </property>
   </widget>
   <widget class="ToggleSwitch" name="toggleSwitchRemoteAlarm" native="true">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>420</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionConnectDA"/>
    <addaction name="actionConnectVaccum"/>
   </widget>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnectDA">
   <property name="text">
    <string>连接采集卡</string>
   </property>
  </action>
  <action name="actionConnectVaccum">
   <property name="text">
    <string>连接真空计</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GaugePanel</class>
   <extends>QWidget</extends>
   <header>GaugePanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RulerTemp</class>
   <extends>QWidget</extends>
   <header>RulerTemp.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToggleSwitch</class>
   <extends>QWidget</extends>
   <header>ToggleSwitch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Monitor.qrc"/>
 </resources>
 <connections/>
</ui>
